ansible/postdeploy.yml

```diff
@@ -6,4 +6,9 @@
 - hosts: ansible
   tasks:
     - include: tasks/installCatalog.yml
-      when: mode == "deploy"
\ No newline at end of file
+      when: mode == "deploy"
+
+- hosts: ansible
+  tasks:
+    - include: tasks/installOpenwhiskCatalog.yml
```

Comment:
it is one repo now - it won't always be one repo, plus we want this to be a model for others so if someone else contributes a repo with packages, we should be able to install them just the same.